xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-test/xwiki-platform-administration-test-pageobjects/src/main/java/org/xwiki/administration/test/po/ThemesAdministrationSectionPage.java

```diff
@@ -64,82 +70,134 @@ public ThemesAdministrationSectionPage()
         super("Themes");
     }
 
+    private List<WebElement> getThemeOptions(WebElement themeInput) {
+        return themeInput.findElements(By.tagName("option"));
+    }
     private List<WebElement> getColorThemeOptions()
     {
-        return iconThemeInput.findElements(By.tagName("option"));
+        return getThemeOptions(colorThemeInput);
+    }
+    private List<WebElement> getIconThemeOptions()
+    {
+        return getThemeOptions(iconThemeInput);
     }
 
     private List<WebElement> getColibriThemeOptions()
     {
-        return iconThemeInput.findElements(By.xpath("//optgroup[@label='Colibri Themes']//option"));
+        return colorThemeInput.findElements(By.xpath("//optgroup[@label='Colibri Themes']//option"));
     }
     private List<WebElement> getFlamingoThemeOptions()
     {
-        return iconThemeInput.findElements(By.xpath("//optgroup[@label='Flamingo Themes']//option"));
+        return colorThemeInput.findElements(By.xpath("//optgroup[@label='Flamingo Themes']//option"));
     }
 
-    /**
-     * @return the list of available color themes
-     */
-    public List<String> getColorThemes()
+    
+    private List<String> getThemes(WebElement themeInput)
```

Comment:
I see there's a `getColorThemes()` below, what's the difference? Should this method be named `getThemeOptions()` instead?

Reply:
BTW do we use `Theme` or `ColorTheme` vocabulary?